text/0092-replay-issue-creation.md

```diff
@@ -17,43 +17,66 @@ These issues can only be detected on the SDK. The Replay back-end will never hav
 
-When the SDK encounters a "replay issue" it will make an HTTP request to a generic interface which will handle the issue creation process.
+When the SDK encounters a "slow click" it will use `sentry.captureException(slowClick)` to create an error issue on the Issues platform.
 
 **Pros:**
 
-1. We can be tested in isolation without impacting existing production services.
-2. We can expose these new features in a more product agnostic way.
-   - This would allow us to reach much larger customer segments when compared to Replay customers.
+- Significantly larger customer reach (i.e. all javascript SDK customers).
+- Opportunity to upsell customers on Session Replay from a "slow click" event.
+- No code changes required on the Session Replay back end.
+- No code changes required by the Issues team.
 
 **Cons:**
 
-1. Uses errors quota.
-2. Unclear if we're able to use dynamic thresholds.
-   - E.g. "Experienced 10 occurences in the past hour".
-3. The replay containing the issue could not be sampled.
+- Requires SDK changes.
+- Can not change issue sampling parameters without an SDK update.
+  - Or can you?
+  - What if we sample within the error processing pipeline on the back-end?
+  - If error processing is functionally immutable (i.e. there is no organizational will-power to allow product teams to sample within the error pipeline) then we will not be able to sample on the back-end.
 
-### Option 2: SDK Generates Issues Through a Session-Replay Specific Interface
+### Option 2: SDK Creates Issues Through an Issues HTTP Endpoint
 
-The SDK publishes a "replay issue" to the Replay back-end. The back-end will decide to process the issue or not.
+When the SDK encounters a "slow click" it will use `http.post("/issues", data=slowClick)` to create an error issue on the Issues platform.
 
 **Pros:**
 
-1. Does not use errors quota.
-2. Overhead is low.
-   - Publishing to a Kafka consumer can happen asynchronously.
-3. We can use dynamic thresholds.
-   - E.g. "Experienced 10 occurences in the past hour".
-4. Replay is guaranteed to be sampled.
+- Significantly larger customer reach (i.e. all javascript SDK customers).
+- Opportunity to upsell customers on Session Replay from a "slow click" event.
+- No code changes required on the Session Replay back end.
+- We can sample slow click issues on the backend.
+  - This would require buy-in and coordination from the Issues team.
+  - There is nothing preventing us from looking at an HTTP request and making a go/no-go decision.
 
 **Cons:**
 
-1. Poor rollout could impact service availability during testing period.
-2. Requires coordination between the SDK and Ingest to create new issue types.
+- Requires SDK changes.
+- Requies code changes by the Issues team to create a generic interface for creating issues.
+  - This would likely disrupt our June 18 deadline.
+  - Unless the Issues team has excess capacity and a willingness to work on it immediately.
```

Comment:
it also seems like we'd also want to do this in Relay -- we try not to have python endpoints be hit by SDKs. seems like it would be a large amount of work.